src/v2/view-builder/views/ov/NumberChallengePhoneView.js

```diff
@@ -0,0 +1,35 @@
+import { View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+
+let correctAnswer = '';
+const NumberChallengePhoneView = View.extend({
+  initialize () {
```

Comment:
do we need the initialize? can we just say `return this.options.appState.get('currentAuthenticator').contextualData.correctAnswer` in `getTemplateData`?

Reply:
Good call !